README.md

```diff
@@ -252,7 +252,7 @@ import preval from 'babel-plugin-preval/macro'
 const one = preval`module.exports = 1 + 2 - 1 - 1`
 ```
 
-> You could also use [`preval.macro`][preval.macro] if you'd prefer to type less 😀
+> You could also use [`preval.macro`][preval.macro] if you'd prefer to type more 😀
```

Comment:
```javascript
import preval from 'preval.macro'
```

Is shorter than

```js
import preval from 'babel-plugin-preval/macro'
```

So I meant less 😉

Reply:
lol I see. 

I was thinking

```js
preval`/* Some code */`
```

is shorter than

```js
preval.macro` /* Some code */`
```